index.bs

```diff
@@ -461,6 +462,13 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
         thousands of samples of audio per second.<br>
         44100 <code>Hz</code> is equivalent to 44.1 <code>kHz</code>.
       </p>
+
+      <p>
+        The <dfn for='AudioConfiguration' dict-member>spatialRendering</dfn> member
```

Comment:
Slight re-write to ref some other the parts of the spec and hopefully make it more clear:

`The <dfn for='AudioConfiguration' dict-member>spatialRendering</dfn> member indicates that the audio SHOULD be renderered spatially. The details of spatial rendering SHOULD be inferred from the {{AudioConfiguration/contentType}}. If not <a>present</a>, the UA MUST presume spatialRendering is not required. When <code>true</code>, the user agent SHOULD only report this configuration as <a for=MediaCapabilitiesInfo>supported</a> if it can support spatial rendering without falling back to a non-spatial mix of the stream.  `

Reply:
Thanks - I've updated the spec with your description.